From 65ba2617a00635da8a6ae96a7073222ce9b741ab Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Tue, 22 Dec 2015 13:43:52 +0800
Subject: [PATCH 0790/5242] MLK-12046-2 ARM: dts: imx6sl-evk.dts: add elan
 touch support

commit  cacf3c7ca4ad2e527216cba9f91401a47df13e06 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add elan touch screen support on imx6sl-evk board.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |   11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 74ae571..1e16ef0 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -320,6 +320,17 @@
 		};
 	};
 
+	elan@10 {
+		compatible = "elan,elan-touch";
+		reg = <0x10>;
+		interrupt-parent = <&gpio2>;
+		interrupts = <10 2>;
+		gpio_elan_cs = <&gpio2 9 0>;
+		gpio_elan_rst = <&gpio4 4 0>;
+		gpio_intr = <&gpio2 10 0>;
+		status = "okay";
+	};
+
 	mma8450@1c {
 		compatible = "fsl,mma8450";
 		reg = <0x1c>;
-- 
1.7.9.5

